website/content/en/docs/deployment/cognito-rds-s3/guide.md

```diff
@@ -31,13 +32,20 @@ Refer to the [general prerequisites guide]({{< ref "/docs/deployment/prerequisit
 Enable culling for notebooks by following the [instructions]({{< ref "/docs/deployment/configure-notebook-culling.md#" >}}) in configure culling for notebooks guide.
 
 2. Deploy Kubeflow.
+
+    1. Export your pipeline-s3-credential-option
+    ```bash 
+    export PIPELINE_S3_CREDENTIAL_OPTION=<IRSA/STATIC>
```

Comment:
Double checking that this should be upper case since it's lower case everywhere else

Reply:
Similar to deployment_option and installation_option they are capitalized in our docs.